lib/blob-store/index.js

```diff
@@ -9,12 +9,14 @@ import { LiveDownload } from './live-download.js'
 
 // prop = blob type name
 // value = array of blob variants supported for that type
-export const SUPPORTED_BLOB_VARIANTS = /** @type {const} */ ({
+const SUPPORTED_BLOB_VARIANTS = /** @type {const} */ ({
   photo: ['original', 'preview', 'thumbnail'],
   audio: ['original'],
-  video: ['original']
+  video: ['original'],
 })
 
+export { SUPPORTED_BLOB_VARIANTS }
```

Comment:
for posterity, needed to export on a separate line because the jsdoc type cast syntax doesn't play nicely with the export keyword